packages/client/auth-core/src/contexts/authlogin.tsx

```diff
@@ -4,46 +4,44 @@ import AuthModal from "@/components/AuthModal";
 import { CrossmintServiceFactory } from "@/services/CrossmintService";
 import { type ReactNode, createContext, useContext, useEffect, useMemo, useState } from "react";
 
+import { Crossmint } from "@crossmint/common-sdk-base";
+
+export function getCachedJwt(): string | undefined {
+    if (typeof document === "undefined") {
+        return undefined; // Check if we're on the client-side
```

Comment:
To consider: should we drop a log here if so? Or even consider throwing an error if we're certain thats omeone calling this on the backend is screwing up?

Reply:
Leaving for now, tracking [here](https://linear.app/crossmint/issue/WAL-2742/investigate-and-handle-devs-accidentally-running-on-server)